docs/modules/user-guide/partials/proc_devfile-properties.adoc

```diff
@@ -0,0 +1,258 @@
+[id="proc_devfile-properties_{context}"]
+= Devfile properties
+
+[role="_abstract"]
+This page quickly introduces you to devfile properties. The code snippets help you get started developing your own devfile projects. The tables show you what the different devfile properties require if you use these properties. See the following table of contents to go to the devfile property you want to work with:
+
+== SchemaVersion
+
+[source,yaml]
+----
+v2.0
+---
+schemaVersion: 2.0.0
+----
+
+[cols="1,1"]
+|===
+|Key
+|schemaVersion
+
+|Type
+|string
+
+|Required
+|yes
+
+|Description
+|v2.0.0 of the schema
+|===
+
+For more information on the schemaVersion in devfiles, go to: xref:migrating-schema-version.adoc[].
+
+== Metadata
+
+[source,yaml]
+----
+v2.0
+---
+metadata:
+  name: devfile-sample
+  version: 2.0.0
+----
+
+[cols="1,1"]
+|===
+|Key
+|metadataObject
+
+|Type
+|metadataObject
+
+|Required
+|no
+
+|Description
+|metadata to be used
+|===
+
+For more information on the metadata in devfiles, go to: xref:adding-projects-to-a-devfile.adoc[].
```

Comment:
```suggestion
For more information on the metadata in devfiles, see xref:adding-projects-to-a-devfile.adoc[].
```
Also, please check the rendered output to verify that the square brackets resolve the topic name.

Reply:
For examples of cross-references, see the IBM Style Guide section, "Citations and references."

Reply:
Yep, it looks good rendered (see screenshot) and I changed `go to` to `see` thanks! 
<img width="1309" alt="Screen Shot 2021-05-11 at 11 51 15 AM" src="https://user-images.githubusercontent.com/70717303/117846362-5dc0a980-b24f-11eb-83d0-0603790313ed.png">